src/gui/tracks/TrackLabelButton.cpp

```diff
@@ -60,8 +63,8 @@ TrackLabelButton::TrackLabelButton( TrackView * _tv, QWidget * _parent ) :
 	else
 	{
 		setFixedSize( 160, 29 );
-		m_renameLineEdit->move( 30, ( height() / 2 ) - ( m_renameLineEdit->sizeHint().height() / 2 ) );
-		m_renameLineEdit->setFixedWidth( width() - 33 );
+		m_renameLineEdit->move(25, ( height() / 2  - m_renameLineEdit->sizeHint().height() / 2 ) + 1);
```

Comment:
Just thought of this now, is it possible to use layouts to dynamically place the line edit widget instead of calling `move()` explicitly? Sorry for the nitpicks, have to do this.

Reply:
Is there an example of using layouts in the code base? So I can see, how it works?

Reply:
I believe the `MixerChannelView` now uses layout to handle something similar.

Reply:
I will have a look. I need to build for Windows and Mac, too. So I can see if it looks all the same on all systems.

Reply:
@BaraMGB, if you search for `QVBoxLayout`, `QHBoxLayout` or `QBoxLayout` you should find several places in the code. Another type of layout is `QGridLayout`.

I think it might even make sense to add a new widget for the label/line edit combination which implements the switching.

Reply:
@michaelgregorius  "I think it might even make sense to add a new widget for the label/line edit combination which implements the switching."

What do you mean by that?

Reply:
It would be a widget that has a label and a line edit as children. Depending on the mode it is in (display/editing) it would only show one of them and hide the other. It could use a simple layout with only one widget into which one it switches the active widget. The margins would likely be 0 so that you could insert it into other layouts.

An alternative implementation might be to use something like [QStackedWidget](https://doc.qt.io/qt-6/qstackedwidget.html).

Reply:
Actually, I don't see the benefits of your idea. The current implementation is clean and works perfect. Actually this is only a simple bug fix PR. I don't see, why we should it make more complicated as it is. If someone wants to make the width of the track header dynamic one day, he could do it. But in my opinion, this goes beyond the purpose of this pull request.

Reply:
I believe `TrackContentWidget` already uses a `QHBoxLayout` and calling move() explicitly might have different results depending on the host system.

Reply:
> Actually, I don't see the benefits of your idea. The current implementation is clean and works perfect. Actually this is only a simple bug fix PR. I don't see, why we should it make more complicated as it is. If someone wants to make the width of the track header dynamic one day, he could do it. But in my opinion, this goes beyond the purpose of this pull request.

I think it might be a bit more stable solution than the current one as can be seen by this issue. I agree that it is not in the scope of this PR though. It was just meant as some food for thought.

Reply:
@michaelgregorius I guess, the best option for this, is to use the QLineEdit itself. There should not need a stacked widget, because the QLineEdit has got 2 modes, a label mode and an edit mode. I try to push a new commit in the next days. Thank you for your thoughts.

Reply:
> [...] the QLineEdit has got 2 modes, a label mode and an edit mode.

That sounds even better! :smiley: